Channels.hs

```diff
@@ -62,21 +104,19 @@ humanTimeDiff d
   where minutes = d / 60
         hours = minutes / 60
         days = hours / 24
-        doShow x unit = (show $ truncate x) ++ " " ++ unit
+        doShow x unit = (pack $ show $ truncate x) <> " " <> unit
 
 
-jobset :: DiffChannel -> Maybe String
-jobset channel
+toJobset :: Text -> Maybe Text
+toJobset c
  | c == "nixos-unstable"       = Just "nixos/trunk-combined/tested"
  | c == "nixos-unstable-small" = Just "nixos/unstable-small/tested"
  | c == "nixpkgs-unstable"     = Just "nixpkgs/trunk/unstable"
- | "nixos-" `isPrefixOf` c     = Just $ "nixos/release-" ++ (drop 6 c) ++ "/tested"
+ | "nixos-" `DT.isPrefixOf` c  = Just $ "nixos/release-" <> (DT.drop 6 c) <> "/tested"
  | otherwise                   = Nothing
- where
-   c = dname channel
 
 -- | Takes time since last update to the channel and colors it based on it's age
-diffToLabel :: Either String NominalDiffTime -> Color
+diffToLabel :: Either Text NominalDiffTime -> Color
```

Comment:
I know you didn't write this part, but "success", "warning", and "danger" don't really feel like colors to me, and it seems like it could be a simple sum/enum instead of `Text`